ArduCopter/GCS_Mavlink.cpp

```diff
@@ -1875,6 +1868,30 @@ AP_ServoRelayEvents *GCS_MAVLINK_Copter::get_servorelayevents() const
     return &copter.ServoRelayEvents;
 }
 
+#if ADVANCED_FAILSAFE == ENABLED
+AP_AdvancedFailsafe *GCS_MAVLINK_Copter::get_advanced_failsafe() const
+{
+    return &copter.g2.afs;
+}
+#endif
+
+MAV_RESULT GCS_MAVLINK_Copter::handle_flight_termination(const mavlink_command_long_t &packet) {
+    MAV_RESULT result = MAV_RESULT_FAILED;
+
+#if ADVANCED_FAILSAFE == ENABLED
+    if (super(packet) != MAV_RESULT_ACCEPTED) {
```

Comment:
No, that should be left to AFS. If AFS is handling the termination then it makes no sense to handicap AFS's potential actions. Now at the moment copter is quite happily disarming the motors in AFS, but if you were to force the motor disarm at a higher level then thats compromising copter AFS's ability to nicely do whatever action is required.